main/src/main/java/org/mobilitydata/gtfsvalidator/validator/TimepointTimeValidator.java

```diff
@@ -67,7 +67,7 @@ private boolean isTimepoint(GtfsStopTime stopTime) {
   /**
    * Timepoint without time
    *
-   * <p>Severity: {@code SeverityLevel.WARNING}
+   * <p>Severity: {@code SeverityLevel.ERROR}
```

Comment:
This change is too strict - it will ban 230 feeds at Google.

Did you analyse statistics for this rule?

Reply:
As suggested, this PR has been split into different PRs: one for each notice. As of today, we do not have the statistics yet for GTFS datasets, but will update when we have them.